packages/appium/lib/extension/manifest.js

```
@@ -212,23 +212,13 @@ export class Manifest {
     ];
 
     // add dependencies to the queue
-    await new B((resolve, reject) => {
-      glob(
-        'node_modules/{*,@*/*}/package.json',
-        {cwd: this.#appiumHome, silent: true, absolute: true},
-        // eslint-disable-next-line promise/prefer-await-to-callbacks
-        (err) => {
-          if (err) {
-            reject(err);
-          }
-          resolve();
-        }
-      )
-        .on('error', reject)
-        .on('match', (filepath) => {
-          queue.push(onMatch(filepath));
-        });
-    });
+    for await (const filepath of globIterate('node_modules/{*,@*/*}/package.json', {
+      cwd: this.#appiumHome,
```

I assume we still need it to swallow possible error?